modules/phase_field/test/tests/feature_volume_vpp_test/tests

```diff
@@ -21,4 +21,40 @@
     min_parallel = 2
     prereq = 'centroid_output_test'
   [../]
+
+  [./percolation_test]
+    type = 'CSVDiff'
+    input = 'percolation_test.i'
+    csvdiff = 'percolation_test_out_features_0000.csv'
+
+    issues = "#13169"
+    requirement = "The FeatureVolumeVectorPostprocessor shall output whether a percolated pathway exists between
+                    specified primary_percolation_boundaries and secondary_percolation_boundaries."
+    design = "/FeatureVolumeVectorPostprocessor.md"
+  [../]
+
+  [./percolation_test_parallel]
+    type = 'CSVDiff'
+    input = 'percolation_test.i'
+    csvdiff = 'percolation_test_out_features_0000.csv'
+
+    issues = "#13169"
+    requirement = "The FeatureVolumeVectorPostprocessor shall output whether a percolated pathway exists between
+                    specified primary_percolation_boundaries and secondary_percolation_boundaries."
+    design = "/FeatureVolumeVectorPostprocessor.md"
+
+    min_parallel = 2
+    prereq = 'percolation_test'
+  [../]
+
+  [./percolation_test_exodus]
+    type = 'Exodiff'
+    input = 'percolation_test.i'
```

Comment:
Oops! This will be a race condition since it doesn't have a prereq and writes out the same output file. We have a little known feature in the test harness where you can say that you "don't want to delete output files before running and that you don't really want to execute the test" - that's really handy for cases like this where you want to check the CSV and the Exodus without wasting resources. However, I honestly wouldn't bother with the Exodiff because if you end up in a case where an element is right on the threshold it can either be colored or not causing an Exodiff. Maybe just delete the Exodus test altogether?

Reply:
Yeah I don't think we need the Exodiff test. I debated putting it in because it does make it more convenient to have the Exodus file already there when you want to look at it, but it doesn't really test anything that isn't already tested elsewhere. I'll remove it.